compiler/rustc_feature/src/builtin_attrs.rs

```diff
@@ -506,6 +506,11 @@ pub static BUILTIN_ATTRIBUTES: &[BuiltinAttribute] = &[
         naked_functions, experimental!(naked)
     ),
 
+    gated!(
+        export, Normal, template!(Word), WarnFollowing,
+        EncodeCrossCrate::Yes, experimental!(export)
```

Comment:
```suggestion
        EncodeCrossCrate::No, experimental!(export)
```
The results of `exportable_items` are encoded instead.